### DISCUSSION TITLE
How to check if Podman container is actually running?

### DISCUSSION THREAD

**Discussion by VeniceNerd** [unifi-utilities/unifi-common#281] [Q&A] [answered]
Hey guys,

I installed "on-boot-script"  and "container-common" on my UDM Pro. Placed my "20-container-common.sh" and "30-multicast-relay.sh" into "/mnt/data/on_boot." and my "ifFilter.json" in "/mnt/data/on_boot.d_support".

I then ran the following command to install "multicast-relay":

```
podman run -it -d \
--restart=on-failure:10 \
--name="multicast-relay" \
--network=host \
--mount type=bind,src=/mnt/data/on_boot.d_support,dst=/multicast-relay-config \
-e OPTS="--ifFilter=/multicast-relay-config/ifFilter.json" \
-e INTERFACES="br0 br10 br20 br30 br50" \
docker.io/scyto/multicast-relay
```

<img width="569" alt="144343027-e7ad37b6-ae33-414e-8043-ec99aca525ef" src="https://user-images.githubusercontent.com/25128243/144345872-f33acc36-f95d-4801-8a40-cfebe5444e5f.png">

Followed by these two commands:

```
killall avahi-daemon

podman start multicast-relay
```
<img width="569" alt="144343360-af3042ce-dd5a-4df8-923a-077057719d0b" src="https://user-images.githubusercontent.com/25128243/144345948-df8374a6-d3d6-432f-af9f-62744a58e699.png">

There were no error messages or anything but running Discovery tool on a computer on my VLAN10 does not show any devices from my VLAN50. So it looks like nothing actually happened after I installed the multicast relay.

Are there any steps I can take to troubleshoot this to make sure that everything actually installed correctly and is running?

Really deflated that nothing happened after all the research I did to even come this far. :/

EDIT 1:

I ran "podman ps" and got the following result:
<img width="776" alt="144347563-00012e93-fe88-4a20-86ec-0df136197ad3" src="https://user-images.githubusercontent.com/25128243/144348466-2ef4bc82-4fbb-4112-af9b-66878594d927.png">

I'm not sure if "podman ps" should actually be showing my "multicast-relay" instance or not but if it should then something is definitely wrong.

EDIT 2:

After digging trough some forums I found this command "journalctl -u udm-boot.service" and ran it from the Unifi Shell. Not 100% sure what it does but here is the log result:

<img width="852" alt="Screenshot 2021-12-01 at 21 41 49" src="https://user-images.githubusercontent.com/25128243/144364594-08c0406d-1029-46f7-9bdd-9130b6b5381e.png">

EDIT 3:

Ran just "journalctl" from the Unifi Shell and looked out for "udm-boot.service". Here is the result
<img width="837" alt="Screenshot 2021-12-02 at 09 47 56" src="https://user-images.githubusercontent.com/25128243/144476445-b80f9e98-f657-46d7-bdbf-a3307c542369.png">

**Comment by boostchicken** [comment, marked as answer]
do a podman ps -a, see why the container exited after you get its name then do a podman logs <container name>

**Reply by VeniceNerd** [reply]
I’ll do that as soon as I’m back on my computer. Just to confirm though, is “udm-boot.service” supposed to say “loaded / active / exited” or is that not right and it should say “loaded / active / running”?

**Reply by boostchicken** [reply]
nah, its a one liner, it runs and exists, normal execution there bud.
